internal/v2/fakes2av2/fakes2av2.go

```
@@ -47,7 +55,10 @@ func (s *Server) SetUpSession(stream s2av2pb.S2AService_SetUpSessionServer) erro
 				return err
 			}
 		case *s2av2pb.SessionReq_OffloadPrivateKeyOperationReq:
-			// TODO(rmehta19): Implement fake.
+			resp, err = offloadPrivateKeyOperation(req.GetOffloadPrivateKeyOperationReq(), req.GetLocalIdentity().GetHostname())
+			if err != nil {
+				log.Printf("Fake S2A Service: failed to build SessionResp with OffloadPrivateKeyOperationResp: %v", err)
```

Should we also return the error?